app/views/LocationTracking.js

```diff
@@ -36,30 +42,91 @@ class LocationTracking extends Component {
     super(props);
 
     this.state = {
+      timer_intersect: null,
       isLogging: '',
     };
   }
 
   componentDidMount() {
     BackHandler.addEventListener('hardwareBackPress', this.handleBackPress);
     GetStoreData('PARTICIPATE')
-      .then(isParticipating => {
-        console.log(isParticipating);
-
-        if (isParticipating === 'true') {
-          this.setState({
-            isLogging: true,
-          });
-          this.willParticipate();
-        } else {
-          this.setState({
-            isLogging: false,
-          });
-        }
-      })
-      .catch(error => console.log(error));
+        .then(isParticipating => {
```

Comment:
Can you please ensure your Prettier formatter is running on save?

The indent set in prettier is 2 spaces.

```
tabWidth: 2,
useTabs: false,
```

Reply:
This is not changed by me. May be an issue with some space.